x-pack/plugins/event_log/server/es/cluster_client_adapter.ts

```diff
@@ -586,6 +587,43 @@ export function getQueryBodyWithAuthFilter(
     },
   ];
 
+  if (ids.length) {
```

Comment:
Ah, so, funny story

The test I wrote wasn't passing because the thing it was testing was broken

🙃

Reply:
So this actually fixes the `getActionErrorLogWithAuth` function, making it so that it will now correctly filter down to action entries that match the `ids` passed in. Before this change, it would give you ALL action error logs.